playground/package.json

```diff
@@ -25,12 +25,12 @@
     "@commercetools-frontend/l10n": "18.6.0",
     "@commercetools-frontend/permissions": "18.6.0",
     "@commercetools-frontend/sdk": "18.6.0",
+    "@commercetools-uikit/checkbox-input": "^11.2.0",
```

Comment:
haha I just did this, pushed and got rejected. you beat me to it 😓

Reply:
Yeah I was about to write again but I thought to be easier to just push the fix.